lib/fastlane/plugin/revenuecat_internal/actions/create_next_snapshot_version_action.rb

```diff
@@ -0,0 +1,81 @@
+        branch_name = "bump/#{next_version_snapshot}"
+
+        Helper::RevenuecatInternalHelper.create_and_checkout_new_branch(branch_name)
+
+        Helper::RevenuecatInternalHelper.replace_version_number(previous_version_number,
+                                                                next_version_snapshot,
+                                                                files_to_update,
+                                                                files_to_update_without_prerelease_modifiers)
+
+        Helper::RevenuecatInternalHelper.commmit_changes_and_push_current_branch('Preparing for next version')
+
+        Helper::RevenuecatInternalHelper.create_pr_to_main("Prepare next version: #{next_version_snapshot}", nil, repo_name, github_pr_token)
+      end
+
+      def self.description
+        "Bumps minor version and adds -SNAPSHOT suffix to version. Creates a PR with the changes"
+      end
+
+      def self.authors
+        ["Toni Rico"]
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :current_version,
+                                       description: "Current version of the sdk",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :repo_name,
+                                       env_name: "RC_INTERNAL_REPO_NAME",
+                                       description: "Name of the repo of the SDK",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :github_pr_token,
+                                       env_name: "GITHUB_PULL_REQUEST_API_TOKEN",
+                                       description: "Github token to use to create the release PR",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :files_to_update,
+                                       env_name: "RC_INTERNAL_FILES_TO_UPDATE_VERSION",
+                                       description: "Files that contain the version number and need to have it updated",
+                                       optional: false,
+                                       type: Array),
+          FastlaneCore::ConfigItem.new(key: :files_to_update_without_prerelease_modifiers,
+                                       env_name: "RC_INTERNAL_FILES_TO_UPDATE_VERSION_WITHOUT_PRERELEASE_MODIFIERS",
+                                       description: "Files that contain the version number without release modifiers and need to have it updated",
+                                       optional: true,
+                                       default_value: [],
+                                       type: Array),
```

Comment:
for this specific command, this will effectively mean "files for which you don't want to add the `-SNAPSHOT` suffix, right? 
Maybe we could name this param accordingly?

Reply:
I wanted it to share the name of the similar parameter of the `BumpVersionUpdateChangelogCreatePrAction`. Even though both actions don't do the same thing, the behavior will be similar, so I think it's better to keep it consistent... I could rename it to something like `files_to_update_without_snapshot_suffix` if you think it's better, but I don't know if it's worth the inconsistency.